plugins/intel-gsc/fu-igsc-plugin.c

```diff
@@ -38,9 +38,26 @@ fu_igsc_plugin_constructed(GObject *obj)
 	fu_plugin_add_firmware_gtype(plugin, NULL, FU_TYPE_IGSC_OPROM_FIRMWARE);
 }
 
+static void
+fu_igsc_plugin_device_changed(FuPlugin *plugin, FuDevice *device)
+{
+	// Check for the WEDGED property in device metadata
+	const gchar *wedged = fu_device_get_metadata(device, "WEDGED");
+	if (wedged && g_strcmp0(wedged, "firmware-flash") == 0) {
+		// Prompt user to recover manually
+		g_warning("Detected WEDGED=firmware-flash uevent. Install recovery firmware that "
+			  "is included in the cab under /recovery with:\n/usr/bin/fwuptool install "
```

Comment:
I think ideally we want to recover from this wedged event just setting the version to 0.0 and then the user can "fwupdmgr update" to any version available. I also think we want to keep around the "just installed" .cab archive and make it available "offline" -- although I'm not sure of the best way to do this. Maybe something like /usr/share/fwupd/remotes.d/vendor/firmware but in /var/lib/fwupd/metadata/recovery or something.